### PR TITLE
Add 'Getting started' section to Contributing doc

### LINKED ISSUE #2: Add "Getting Started" section to contributing doc with setup instructions
Just need basic TL;DR setup instructions.

### PR DESCRIPTION
**Pull request by nihonjinrxs**
Resolves #2  

### Description

This adds step by step setup instructions to the Contributing doc.

### Type of change

* Documentation update

### How Has This Been Tested?

Rendered the markdown in IDE and looked at it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,3 +31,23 @@ Some things that will increase the chance that your pull request is accepted:
...
+
+To setup the app, you'll want to do the following:
+
+1. Clone the repo:
+  - SSH: `git clone git@github.com:rubyforgood/coral.git`
+  - HTTPS `git clone https://github.com/rubyforgood/coral.git`
+2. Change to the app directory: `cd coral`
+3. Be sure Postgres is [installed](https://www.postgresql.org/download/) (version 9.1+) and running, or that you've started it locally [in a container](https://hub.docker.com/_/postgres)
+4. If you're using `rvm` or `rbenv` (you should), ensure you've got the right ruby version selected:
+  - RVM: `rvm current` to check version, `rvm use` to change version
```
Comment (cflipse) [inline CONTRIBUTING.md:45]: I am way past the point where I think it's at all acceptable to promote RVM.  It's a bad system, and I don't want to  push it on newbies.

rbenv or chruby  and generally we just need to handle the `.ruby-version` ... which is in the repo; the only check is to make sure that (at current) 2.6.3 is installed.

**Comment by cflipse** [comment]
This looks fine for localdev, to a point.  I _know_ that we're going to end up needing nginx, redis and some kind of storage system ... all of which are going to require separate, and generally OS specific instructions.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nihonjinrxs) [reply CONTRIBUTING.md:45]: Some of us have it already (for years in my case), so I thought it best to just include that. Not promoting RVM necessarily, but happy to add chruby too.

**Comment by nihonjinrxs** [comment]
Yep, idea was just to get folks to the point of being able to work locally. We can add/modify as we include those pieces. Also, for service worker stuff, based on what I've read, https is only needed when you're not on localhost, so for local feature dev, things should be fine without extra complexity.

File: CONTRIBUTING.md (hunk 1 above)
Comment (cflipse) [reply CONTRIBUTING.md:45]: If you've already got RVM, you know how to use it.

Granting it pride of place _is_ advertising it

**Comment by cflipse** [comment]
Ah, managed to skip right over the part where serviceworkers will work fine on localhost.  cool.

**Pull request merged**